### DISCUSSION TITLE
Storage folder restriction

### DISCUSSION THREAD

**Discussion by heitorprojects** [trntv/yii2-starter-kit#783] [General]
Hi,

Can one specify a subfolder of @storage/web as the only folder that users can view and upload content to (eg. @storage/web/uploads)?

Currently my file-manager shows and allows access to everything in @storage/web - including my 'index.php' and '.htaccess' files, which malicious or careless users can access and delete...

Thanks,
Jose

**Comment by XzAeRo** [comment]
hi @heitorprojects,

You can always start by changing the folder permissions accordingly in your server and denying folder/file listing.

Also, you can change the file manager settings in [`backend/modules/file/controllers/ManagerController.php`](https://github.com/yii-starter-kit/yii2-starter-kit/blob/main/backend/modules/file/controllers/ManagerController.php). You can find the documentation about the plugin here: https://github.com/alexantr/yii2-elfinder

**Reply by heitorprojects** [reply]
> Also, you can change the file manager settings in [`backend/modules/file/controllers/ManagerController.php`](https://github.com/yii-starter-kit/yii2-starter-kit/blob/main/backend/modules/file/controllers/ManagerController.php). 

Perfect - Thanks.
